.editorconfig

```diff
@@ -2,7 +2,7 @@ root = true
 
 [*]
 indent_style = tab
-indent_size = 1
+indent_size = 4
```

Comment:
Try `cat`-ing the source and see how many columns the tabs are.

Or try `less`.  Or `git diff`, for that matter. ;)

That's not to say that I prefer tabs for _indentation in source code_.  I _like_ 4-column _indentation_!